src/NUnitFramework/framework/Constraints/CollectionEquivalentConstraint.cs

```diff
@@ -112,5 +112,30 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
             base.Using(EqualityAdapter.For(comparison));
             return this;
         }
+
+
+        /// <summary>
+        /// Flag the constraint to use the supplied Comparison object.
+        /// </summary>
+        /// <param name="comparison">The Comparison object to use.</param>
+        #pragma warning disable CS0612
+        [Obsolete]
+        public new CollectionItemsEqualConstraint Using<T>(Comparison<T> comparison)
+        {
+            Comparer.ExternalComparers.Add(EqualityAdapter.For(comparison));
+            return this;
+        }
+
+        /// <summary>
+        /// Flag the constraint to use the supplied boolean-returning delegate.
+        /// </summary>
+        /// <param name="comparer">The supplied boolean-returning delegate to use.</param>
+        #pragma warning disable CS0612
+        [Obsolete]
+        public new CollectionItemsEqualConstraint Using<T>(Func<T, T, bool> comparer)
+        {
+            Comparer.ExternalComparers.Add(EqualityAdapter.For(comparer));
+            return this;
+        }
```

Comment:
Same as above. I know that these methods are the last in the file, but if somebody at some point adds methods to the end of the class these will also be covered by the `#pragma warning disable CS0612` unless they are restored.